pytensor/tensor/rewriting/linalg.py

```diff
@@ -52,29 +52,28 @@ def inv_as_solve(fgraph, node):
 @node_rewriter([Solve])
 def tag_solve_triangular(fgraph, node):
```

Comment:
```suggestion
def solve_cholesky_to_solve_triangular(fgraph, node):
```

Reply:
Yeah agreed, "tag" doesn't mean anything. But we shouldn't call it "solve_cholesky", because there's a separate `Op` called `SolveCholesky`. Maybe `generic_solve_to_solve_triangular`?

Reply:
Sounds good